2/src/index.html

```diff
@@ -0,0 +1,682 @@
+                fill="white"
+              />
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M32.8313 11.7184L42.535 0.429688H36.7488L29.0971 9.80387H26.9164V0.429688H22.3008V23.5711H26.9164V14.0519H29.047L37.2811 23.5711H43.2648L32.8313 11.7184Z"
+                fill="white"
+              />
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M84.537 0.430054V14.1013C84.537 17.8857 82.3694 19.5711 79.1959 19.5711C76.0225 19.5711 73.8592 17.8857 73.8592 14.1013V0.430054H69.2305V14.3315C69.2305 20.7286 73.3127 23.985 79.1959 23.985C85.0836 23.985 89.1668 20.7286 89.1668 14.3315V0.430054H84.537Z"
+                fill="white"
+              />
+              <mask
+                id="mask0"
+                mask-type="alpha"
+                maskUnits="userSpaceOnUse"
+                x="91"
+                y="0"
+                width="20"
+                height="24"
+              >
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M91.4766 0.430054H110.438V23.5715H91.4766V0.430054Z"
+                  fill="white"
+                />
+              </mask>
+              <g mask="url(#mask0)">
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M91.4766 0.430054V4.74242H98.6504V23.5715H103.265V4.74242H110.439V0.430054H91.4766Z"
+                  fill="white"
+                />
+              </g>
+              <mask
+                id="mask1"
+                mask-type="alpha"
+                maskUnits="userSpaceOnUse"
+                x="42"
+                y="0"
+                width="25"
+                height="24"
+              >
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M42.1484 0.000732422H66.1528V24H42.1484V0.000732422Z"
+                  fill="white"
+                />
+              </mask>
+              <g mask="url(#mask1)">
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M54.1528 19.3426C50.0968 19.3426 46.811 16.0567 46.811 11.9996C46.811 7.94801 50.0968 4.65782 54.1528 4.65782C58.2044 4.65782 61.4913 7.94801 61.4913 11.9996C61.4913 16.0567 58.2044 19.3426 54.1528 19.3426ZM54.1528 0.000732422C47.5212 0.000732422 42.1484 5.37346 42.1484 11.9996C42.1484 18.6269 47.5212 24.0007 54.1528 24.0007C60.7801 24.0007 66.1528 18.6269 66.1528 11.9996C66.1528 5.37346 60.7801 0.000732422 54.1528 0.000732422Z"
+                  fill="white"
+                />
+              </g>
+            </svg>
+          </a>
+        </div>
+
+        <div class="flex-grow-1">
+          <div class="hamburger d-none d-md-block" id="hamburger">
```

Comment:
it should be a button with `span`  inside

Reply:
done